File: NyanSpecialty.Assistance.Web.Models/UserRegistration.cs
namespace NyanSpecialty.Assistance.Web.Models
{
    public class UserRegistration : CommonProps
    {
        public long? Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public string Phone { get; set; }
        public string Password { get; set; }
        public long? CustomerId { get; set; }
        public long? ProviderId { get; set; }
        public long? RoleId { get; set; }
        public DateTimeOffset? LastPasswordChangedOn { get; set; }
        public bool? IsBlocked { get; set; }

    }
}

File: NyanSpecialty.Assistance.Web.UI/src/app/models/userregistration.ts
import { CommonProps } from "./commonprops";

export interface UserRegistration extends CommonProps {
    id: number;
    firstName: string;
    lastName: string;
    email: string;
    phone: string;
    password: string;
    customerId?: number;
    providerId?: number;
    roleId?: number;
    lastPasswordChangedOn?: Date;
    isBlocked?: boolean;
}
